.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,106 @@
+name: Report a bug
+description: Errors and regression reports with complete reproducing test cases.
+labels: ["status:triage", "bug"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Bug reports aren't usually really **🐛 bugs**. More often than not they are questions about how to use the product.
+        Hit us up on our [Community Slack](https://pytorch-lightning.slack.com) first if that is the case.
```

Comment:
I'd suggest GH discussions. Will reduce the prob of repeated questions we get.

Reply:
GH discussions is definitely better in terms of indexing for others but worse in getting notifications for us to answer.

Reply:
we could add a GH notification app on a slack channel for discussions.
but looks like it's not there yet: https://github.com/integrations/slack/issues/1223, https://github.com/integrations/slack/issues/1134

Reply:
Modified this

Reply:
I can't seem to find the discussion link, Still slack link is attached